application-licensing-licensor/application-licensing-licensor-ui/src/main/resources/Licenses/Code/VelocityMacros.xml

```diff
@@ -193,34 +193,41 @@
   {{/html}}
 #end
 
-#macro (licenseButton $licenseType $cssClass $key $jsonData)
+#macro (licenseButton $licenseType $cssClass $iconName $key $jsonData)
   #set ($jsonData.licenseType = $licenseType)
-  #set ($label = $services.localization.render("licensor.${key}License.label"))
-  &lt;button class="btn btn-$cssClass licenseButton licenseButton-$escapetool.xml($key)"
-    data-button="$escapetool.xml($jsontool.serialize($jsonData))"&gt;$label&lt;/button&gt;
+  #set ($label = $escapetool.xml($services.localization.render("licensor.${key}License.label")))
+  &lt;a href="" class="action licenseButton licenseButton-$escapetool.xml($key)"
```

Comment:
```suggestion
  &lt;a href="#$escapetool.xml($key)" class="action licenseButton licenseButton-$escapetool.xml($key)"
```

It's recommended to specify the ``href`` attribute otherwise the link may look different and you may have some accessibility issues.